EIPS/eip-5792.md

```diff
@@ -124,7 +129,10 @@ Note that since the `paymasterService` `capability` is marked as optional, walle
 
 ##### `wallet_sendCalls` Example Return Value
 
-The identifier MUST be unique 64 bytes represented as a hex encoded string.
+If the `id` field was provided by the app, the wallet MUST return the same value in the response.
+
+Otherwise, the identifier MUST be a unique string up to 4096 bytes (8194 characters including leading `0x`).
```

Comment:
I would just remove `Otherwise` as it might imply the dapp has no limitations on the ID while the wallet does:
```
The identifier MUST be a unique string up to 4096 bytes (8194 characters including leading `0x`).  
```